packages/pn-pa-webapp/src/components/Notifications/FilterNotifications.tsx

```diff
@@ -189,8 +189,8 @@ const FilterNotifications = forwardRef(({ showFilters }: Props, ref) => {
     </CustomMobileDialog>
   ) : (
     <form onSubmit={formik.handleSubmit} data-testid="filter-form">
-      <Box
-        display={'flex'}
+      <Stack
```

Comment:
Why this change?

Reply:
isn't it the same?

Reply:
do you have checked all the layouts and all the use cases?

Reply:
yes, all works